javascript/javascript1/week4/voice-assistant.js

```diff
@@ -0,0 +1,103 @@
+let name = [];
+let todoList = [];
+function getReply(command) {
+  if (command.toLowerCase().includes("hello my name is")) {
+    const newName = command.split(" ")[4];
+    const commandWords = command.split(" ");
+    if (commandWords.length >= 5 && name.length > 0 && name.includes(newName)) {
+      return `Hello again, ${newName}`;
+    } else {
+      name.push(newName);
+      return `Nice to meet you ${newName}`;
+    }
+  }
+  if (command.toLowerCase().includes("what is my name")) {
+    return `Your name is ${name}`;
+  }
+  if (
+    command.toLowerCase().includes("add") &&
+    command.toLowerCase().includes("todo")
+  ) {
+    const todoId = command.indexOf("todo");
+    const todoItem = command.split(" ")[1];
+    todoList.push(todoItem);
+    return `${todoItem} added to your todo`;
+  }
+  if (
+    command.toLowerCase().includes("remove") &&
+    command.toLowerCase().includes("todo")
+  ) {
+    const todoId = command.indexOf("todo");
+    const todoItem = command.split(" ")[1];
+    todoList.splice(todoList.indexOf(todoItem), 1);
+    return `${todoItem} removed from your todo`;
+  }
+  if (command.toLowerCase().includes("what is on my todo?")) {
+    return `You have ${todoList.length} todos - ${todoList}`;
+  }
+  if (command.toLowerCase().includes("what day is it today?")) {
+    const today = new Date();
+    const date = today.getDate();
+    const month = today.getMonth() + 1;
+    const year = today.getFullYear();
+    return `${date}.${month}.${year}`;
+  }

```

Comment:
Almost! The homework says: _if today is 30/8/2019 then it should respond with 30. of August 2019_